python/example_code/ec2/ec2_instance_management.py

```diff
@@ -16,6 +16,7 @@
 ec2 = boto3.resource('ec2')
 
 
+# snippet-start:[python.example_code.ec2.StartInstances]
 def start_instance(instance_id):
     """
     Starts an instance. The request returns immediately. To wait for the instance
```

Comment:
Should one say
Amazon EC2 instance (instance)
on first use? And just use instance thereafter?

Reply:
Good question: on the page where this snippet is shown, the description text above the snippet reads "The following code example shows how to start an Amazon EC2 instance."